README.md

```diff
@@ -92,55 +66,58 @@ If you want to contribute to this project and do not know where to start [good f
 
 2. Install global dependencies:
```

Comment:
hey @bryceosterhaus, I think we can remove those steps and just point to the [CONTRIBUTING.md](https://github.com/liferay/clay/blob/develop/CONTRIBUTING.md).

We can put the information here for those who want to contribute with version v2, something like this:
```
### Contributing to Clay v2

For contributions with Clay v2, go to branch v2 and follow the [contribution guide](https://github.com/liferay/clay/blob/2.x/CONTRIBUTING.md).
```